.github/workflows/build.yml

```diff
@@ -20,11 +20,15 @@ jobs:
       uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
 
     - name: Install dependencies
-      run: python -m pip install jupyterlab~=3.0
+      run: python -m pip install jupyterlab~=3.0 jupyter-server==1.23.3
```

Comment:
Is this tracked somewhere why Jupyter Server 2 is causing issues?

Reply:
https://github.com/jupyter-server/jupyter_server/issues/1134

Reply:
After the PR is merged we should also open an issue in this repo to keep track of removing this pin on CI.

Reply:
The thing is that if we first install JupyterLab, we get `jupyter-server` v2 and the `jupyter-server-terminals` extension which is incompatible with `jupyter-server` v1.
I see that `jupyter-resource-usage` requires `jupyter-server` v1, maybe we should also allow v2?

Reply:
>  I see that jupyter-resource-usage requires jupyter-server v1, maybe we should also allow v2?

Yes sounds good :+1: